easybuild/easyblocks/i/inspector.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, *args, **kwargs):
 
         # recent versions of Inspector are installed to a subdirectory
         self.subdir = ''
-        if LooseVersion(self.version) >= LooseVersion('2013_update7') and LooseVersion(self.version) < LooseVersion('2017'):
+        if LooseVersion(self.version) >= LooseVersion('2013_update7') and \
+           LooseVersion(self.version) < LooseVersion('2017'):
```

Comment:
Ugh, not a big fan of the break via `\` (we don't do this anywhere else either).

If you want to shorten this, I'd suggest:

```python
loosever = LooseVersion(self.version)
if loosever >= LooseVersion('2013_update7') and loosever < LooseVersion('2017'):

Reply:
Ok. I've just done it like that because the vtune easyblock had that break in that point.